.drone.jsonnet

```diff
@@ -316,7 +316,7 @@ local static_build(name,
   debian_pipeline(
     'Static Android',
     docker_base + 'android',
-    deps=['google-android-ndk-installer'],
+    deps=['google-android-ndk-r25c-installer'],
```

Comment:
You ought to be able to just drop this line, and rely on whatever is in the `android` docker image (which currently is the r25c version).  That way we only have to update it there when it changes rather than multiple repos.

Reply:
(It's installed here: https://github.com/oxen-io/oxen-docker/blob/bf4571505aef62010e695daaff4239ee8905731c/ci/rebuild-docker-images.py#L321)